Update README.md (#2330)
viamrobotics/docs: README.md

### Lint JS and Markdown files with Prettier on save

1. Install the [Prettier VS Code Extension](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode).
2. Run `npm install` in the docs folder where you have the docs checked out.

> 3. Inside VS code, open user `settings.json` by pressing `CMD+SHIFT+P` and typing in settings, and ensure the following settings are in the file:

Reviewer: ```suggestion
3. Inside VS code, open the _user_ `settings.json` file by pressing `CMD+SHIFT+P` and typing 'settings', and add the following settings to the end of the file:
```
